From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Thu, 30 Jul 2026 01:54:29 +0200
Subject: [PATCH 091/109] fixup! - Move root to /usr. - Change sorting mount
 points. - By default mount without ACLs. - Can read /etc/fstab with short
 mount point format.

(cherry picked from commit feae3514dfefb190c990cd432e72888e0883aa29)
(cherry picked from commit 6412890294923a167f706b3c28e5f92551c3a6d0)
---
 winsup/testsuite/cygrun.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/winsup/testsuite/cygrun.sh b/winsup/testsuite/cygrun.sh
index bf1d5cc6b..5479543ab 100755
--- a/winsup/testsuite/cygrun.sh
+++ b/winsup/testsuite/cygrun.sh
@@ -11,7 +11,7 @@ export PATH="$runtime_root:${PATH}"
 if [ "$1" = "./mingw/cygload" ]
 then
     windows_runtime_root=$(cygpath -m $runtime_root)
-    $cygrun "$exe -v -cygwin $windows_runtime_root/cygwin1.dll"
+    MSYS_NO_PATHCONV=1 $cygrun "$exe -v -cygwin $windows_runtime_root/msys-2.0.dll"
 else
     cygdrop $cygrun $exe
 fi
